google-cloud-clients/google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/StorageRpc.java

```diff
@@ -282,6 +284,14 @@ StorageObject compose(
    */
   Tuple<String, byte[]> read(StorageObject from, Map<Option, ?> options, long position, int bytes);
 
+  /**
+   * Reads from a storage object at the given position directly to outputstream
```

Comment:
"Reads ...to the output stream. "If you want to use outputstream as one word, do it consistently across. in line 293, you have used it as outputStream, here it is outputstream. However, I feel the meaning still stands when you use output stream instead.